mungegithub/features/repo-updates.go

```diff
@@ -287,14 +291,14 @@ func peopleForPath(path string, people map[string]sets.String, leafOnly bool, en
 	return out
 }
 
-// LeafAssignees returns a set of users who are the closest assginees to the
+// LeafAssignees returns a set of users who are the closest assignees to the
 // requested file. If pkg/OWNERS has user1 and pkg/util/OWNERS has user2 this
 // will only return user2 for the path pkg/util/sets/file.go
 func (o *RepoInfo) LeafAssignees(path string) sets.String {
 	return peopleForPath(path, o.assignees, true, o.EnableMdYaml)
```

Comment:
They are merged together. Although under the "old" name `o.assignees`.  This patch LGTM, it is backwards compatible...